demosplan/DemosPlanCoreBundle/Controller/Statement/DemosPlanStatementController.php

```diff
@@ -1669,8 +1672,12 @@ protected function replacePhaseByPhaseNameForVotedStatementList(array $statement
      *
      * @throws Exception
      */
-    protected function createPdfDraftStatement($draftStatementList, $type, Procedure $procedure, $itemsToExport = null)
-    {
+    protected function createPdfDraftStatement(
```

Comment:
You need to modify the method calls as well when you change the signature

Reply:
sorry fixed here https://github.com/demos-europe/demosplan-core/pull/1573/commits/058f6f64501dd13389225679e170f073ed202a6e